File: GKPageScrollViewDemo/GKPageScrollViewDemo/GKPageScrollView/GKPageTableView.m
//
//  GKPageTableView.m
//  GKPageScrollView
//
//  Created by QuintGao on 2018/10/26.
//  Copyright © 2018 QuintGao. All rights reserved.
//

#import "GKPageTableView.h"

@implementation GKPageTableView

// 允许多个手势
- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer {
    return YES;
}

@end

File: GKPageScrollView/swift/GKPageTableView.swift
//
//  GKPageTableView.swift
//  GKPageScrollViewSwift
//
//  Created by gaokun on 2019/2/20.
//  Copyright © 2019 gaokun. All rights reserved.
//

import UIKit

open class GKPageTableView: UITableView, UIGestureRecognizerDelegate {
    
    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
        return true
    }
}
